Framework/include/QualityControl/QcInfoLogger.h

```diff
@@ -66,6 +67,10 @@ class QcInfoLogger : public AliceO2::InfoLogger::InfoLogger
   // Disallow copying
   QcInfoLogger& operator=(const QcInfoLogger&) = delete;
   QcInfoLogger(const QcInfoLogger&) = delete;
+
+  // remember the contexts
+  AliceO2::InfoLogger::InfoLoggerContext* mContext = nullptr;
+  AliceO2::InfoLogger::InfoLoggerContext* mDplContext = nullptr;
```

Comment:
How about making our context a `shared_ptr` so it is deleted?
I would leave the DPL context as it is though, because we only want to observe it, I assume.

Reply:
agreed, on it